Triangulation/include/CGAL/Delaunay_triangulation.h

```diff
@@ -779,10 +780,11 @@ typename Delaunay_triangulation<DCTraits, TDS>::Vertex_handle
 Delaunay_triangulation<DCTraits, TDS>
 ::insert_in_conflicting_cell(const Point & p, const Full_cell_handle s)
 {
+    // for storing conflicting full_cells.
     typedef std::vector<Full_cell_handle> Full_cell_h_vector;
-    static Full_cell_h_vector cs; // for storing conflicting full_cells.
+    CGAL_STATIC_THREAD_LOCAL_VARIABLE(Full_cell_h_vector,cs,0);
     cs.clear();
-    // cs.reserve(64);
```

Comment:
(Not that the call was already commented.)

The reserve is useless since the variable was flagged `static`: the vector `capacity()` is preserve from one call to the function to another, only the size is reset to 0, with `cs.clear()`.